build.ps1

```diff
@@ -31,8 +31,36 @@ function test() {
 function publish() {
     Remove-Item ./drop -Force -Recurse -ErrorAction Ignore
     exec "dotnet pack src\docfx -c Release -o $PSScriptRoot\drop /p:Version=$version /p:InformationalVersion=$version"
+    if ($env:BUILD_SOURCEBRANCH -eq "refs/heads/v3-release")
+    {
+        publishLocalBuildPackage
+    }
 }
 
+function publishLocalBuildPackage() {
+    $localBuildPackagePath = "$PSScriptRoot\drop\local-build-packages"
+    $blobUrl = "https://opbuildstorageprod.blob.core.windows.net/docfx-local-build-packages/"
+    $windowsRuntime = "win-x64"
+    $osxRuntime = "osx-x64"
+    Remove-Item $localBuildPackagePath -Force -Recurse -ErrorAction Ignore
```

Comment:
can publish to a different staging folder to avoid deleting here. The files might be useful for troubling so keep it here.